lib/arjdbc/postgresql/adapter.rb

```diff
@@ -52,8 +52,15 @@ def postgresql_version
       @postgresql_version ||=
         begin
           version = @connection.database_product
-          if version =~ /PostgreSQL (\d+)\.(\d+)\.(\d+)/
-            ($1.to_i * 10000) + ($2.to_i * 100) + $3.to_i
+          if version.match /PostgreSQL ((\d+\.)+\d+)/
+            version_numbers = $1.split('.').map(&:to_i)
```

Comment:
match does not use `$1`, thus this seems self invented (not AR core port). 
maybe its better to have what Rails is using or if its different at least a unit test for the 'compicated' versioning part?
also did this get fixed in AR 5.0 or 5.1?